### PR TITLE
fix referral duplicates

### PR DESCRIPTION
**Pull request by 0xRobin**
fix referral duplicates

### CODE REVIEW

**Comment by 0xRobin** [comment]
The core of this issue was a bas price token. 

This is fixed in this PR: https://github.com/duneanalytics/spellbook/pull/6262

We should be able to merge this once that is merged and `prices.usd` is updated

**Comment by jeff-dude** [comment]
to be safe, i will let prices run in prod first and then have CI run here

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**
